src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/BookmarksModal.scss

```diff
@@ -31,7 +31,8 @@
 
 .more {
   margin-top: 25px;
-  span { /* stylelint-disable-line no-descending-specificity */
+  span {
+    /* stylelint-disable-line no-descending-specificity */
```

Comment:
Does stylelint complain if we remove this line?

Reply:
Yes it still does:
```src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/BookmarksModal.scss
 34:3  ⚠  Expected selector ".more span" to come before selector ".linkHolder .label span + span"  no-descending-specificity```